### DISCUSSION TITLE
Allow drag to higher UAC application

### DISCUSSION THREAD

**Discussion by WIZZsWIZZs** [DelphiPraxis/The-Drag-and-Drop-Component-Suite-for-Delphi#68] [Ideas]
Hi,
Is there a way to allow dragging a file with TDropFileTarget from Windows Explorer to an elevated Delphi application (run as Administrator)?
Thanks.

**Comment by landrix** [comment]
No, it is not possible

https://github.com/landrix/The-Drag-and-Drop-Component-Suite-for-Delphi/issues/43

**Comment by WIZZsWIZZs** [comment]
According to this post it can be done.
[https://stackoverflow.com/questions/57976627/delphi-drag-and-drop-files-from-windows-explorer-to-tlistview-does-not-work](https://github.com/landrix/The-Drag-and-Drop-Component-Suite-for-Delphi/issues/url)
Therefor I was hoping it could be applied to the component as well.
